cypress/e2e/code-of-conduct.cy.js

```diff
@@ -0,0 +1,101 @@
+describe('Tests Mentorship Code of Conduct page', () => {
+    beforeEach(() => {
+      cy.visit(`${Cypress.config().baseUrl}/mentorship-code-of-conduct`);
+    });
+  
+    describe('Verify UI elements from Header and Main content', () => {
+  
+      it('should verify UI elements on the Code of Conduct page', () => {
+        cy.contains("Mentorship Code of Conduct").should('be.visible');
+        cy.contains("Code Of Conduct for Mentees").should('be.visible');
+        cy.contains("Code Of Conduct for Mentors").should('be.visible');
+        cy.contains('Show More').should('be.visible');
```

Comment:
```suggestion
        cy.contains('Show More').shouldBeVisible();
```

I think you can use like this.

Reply:
you can do this in another PR ok? I guess it will be nice to see how this will improve the code readability.

Reply:
I agree,  I will take care of this in another PR.